oak_functions/lookup_data_checker/Cargo.toml

```diff
@@ -14,5 +14,4 @@ location_utils = { path = "../location_utils" }
 log = "*"
 oak_functions_abi = { path = "../abi" }
 prost = "*"
-rand = "*"
 serde = { version = "*", features = ["derive"] }
```

Comment:
A little bit confused why this PR needs to update this. It appears as that this unused dep is an issue that precedes these changes, but was flagged when running CI locally.

Reply:
cargo-udeps was disabled on the server CI for a while. I am not sure whether it has been re-enabled.